module/os/linux/zfs/zfs_ioctl_os.c

```diff
@@ -135,7 +135,7 @@ zfsdev_ioctl(struct file *filp, unsigned cmd, unsigned long arg)
 
 	vecnum = cmd - ZFS_IOC_FIRST;
 
-	zc = kmem_zalloc(sizeof (zfs_cmd_t), KM_SLEEP);
+	zc = vmem_zalloc(sizeof (zfs_cmd_t), KM_SLEEP);
```

Comment:
[nit] do we want to pair with a `vmen_free()` for symmetry?  They are both backed by `spl_vmem_free_impl` so no net difference.

Reply:
good call.. done.